src/Diagnostics.DataProviders/DataProviderLogDecorator.cs

```diff
@@ -61,215 +61,215 @@ public DataProviderMetadata GetMetadata()
 
         #region AppInsight_DataProvider
 
-        public Task<DataTable> ExecuteAppInsightsQuery(string query)
```

Comment:
Please revert this file

Reply:
If I revert this file, the loading of detectors will timeout.

Reply:
This is the same code in production and we don't run into this. Lets sync up.